test/python/tests/nexus/test_multi_nexus.py

```diff
@@ -126,6 +127,7 @@ async def mount_devices(connect_devices):
 
     for dev in connect_devices:
         await run_cmd_async(f"sudo umount /mnt{dev}")
+    await run_cmd_async(f"sudo rm -rf /mnt/dev")
```

Comment:
Is this actually `/mnt/dev` or `/mnt{dev}` , the directory that's unmounted above?

Reply:
They're all in /mnt/dev, eg: /mn/dev/nvme**, so if we remove /mnt/dev we remove them all